### PR TITLE
Safety: assert that we use the steering req bit

### PR DESCRIPTION
**Pull request by jnewb1**
TODO:

- [x] verify all of these and split into PR for each
- [ ] ~~VW needs to control this bit for fault avoidance, any others?~~

https://github.com/commaai/panda/issues/1621

### CODE REVIEW

**Comment by jnewb1** [comment]
just missing vw now

**Comment by sshane** [comment]
example failure catching a safety mode not setting steer_req

```python
======================================================================
FAIL: test_steer_req_bit (__main__.TestGmCameraSafety.test_steer_req_bit)
Asserts all torque safety modes check the steering request bit
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/batman/openpilot/panda/tests/safety/common.py", line 300, in test_steer_req_bit
    self.assertFalse(self._tx(self._torque_cmd_msg(self.MAX_TORQUE, 0)))
AssertionError: 1 is not false
```

**Comment by sshane** [comment]
Changed the test to not modify the safety code, think it's better not to add a variable that's not used anywhere in the safety -- as it's more likely prone to bugs (was missing from the init/reset function).

File: tests/safety/common.py [hunk 1]
```diff
@@ -283,6 +285,22 @@ def test_non_realtime_limit_up(self):
...
 
+  def test_steer_req_bit(self):
+    """Asserts all torque safety modes check the steering request bit"""
+    if self.NO_STEER_REQ_BIT:
+      raise unittest.SkipTest("No steering request bit")
+
+    self.safety.set_controls_allowed(True)
+    self._set_prev_torque(self.MAX_TORQUE)
+
+    # Send torque successfully, then only drop the request bit and ensure it's blocked
```
Comment (sshane) [inline tests/safety/common.py:296]:
```suggestion
    # Send torque successfully, then only drop the request bit and ensure it stays blocked
```

**Pull request merged**